conf.py

```diff
@@ -241,9 +241,21 @@
     {
         "name": "nifi",
         "versions": [
-            {"product": "1.21.0", "java-base": "11"},
-            {"product": "1.23.2", "java-base": "11"},
-            {"product": "1.25.0", "java-base": "21"},
+            {
+                "product": "1.21.0",
+                "java-base": "11",
+                "java-devel": "11"
+            },
+            {
+                "product": "1.23.2",
+                "java-base": "11",
+                "java-devel": "11"
+            },
+            {
+                "product": "1.25.0",
+                "java-base": "21",
+                "java-devel": "11"
```

Comment:
@Jimvin Didn't you spend some time looking at which Java versions NiFi builds with?
Malte is wondering why Java 11 is required to build.

Reply:
I have built NiFi using 11 and 17 in the past. DoesI think that the allowed range of 1.8.0-251,21 excludes 21.0.3. That could be bumped up and tested.